ILSpy.AddIn/Samples/ILSpyAddInSamples.cs

```diff
@@ -0,0 +1,420 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+/// Sample source file indicating a wide variety of code elements that should work
```

Comment:
Each item includes its "/navigateTo" string as a comment. Might be worth someone putting some of these into the documentation for the ILSpy command line option, rather than just saying "The syntax of ID strings is described in appendix A of the C# language specification."